### ISSUE TITLE
Update function not working.

### ISSUE THREAD

**Issue by mvdbent** [txhaflaire/JamfCheck#10]
App starts downloading, looks like he does all the stuff whats needed to been done. 
App gets closed, but after restart you get again a update message.

**Comment by txhaflaire** [comment]
Thank you for reporting in this issue @mvdbent.

While quickly looking into this, this issue is caused by the fact that the initial logic is looking at the first index of the response we get from the "latest" response. Since a .pkg got added it's not the first index anymore.

It will be resolved in the upcoming release, to fix this for existing versions the .pkg will be removed from the latest version for now. 

```
    func getLatestReleaseURL(completion: @escaping (String?) -> Void) {
        let apiURL = URL(string: "https://api.github.com/repos/txhaflaire/JamfCheck/releases/latest")!
        
        URLSession.shared.dataTask(with: apiURL) { data, _, error in
            if let data = data {
                do {
                    if let json = try JSONSerialization.jsonObject(with: data, options: []) as? [String: Any],
                       let assets = json["assets"] as? [[String: Any]],
                       let firstAsset = assets.first,
                       let assetURL = firstAsset["browser_download_url"] as? String {
                        completion(assetURL)
                    } else {
                        completion(nil)
                    }
                } catch {
                    completion(nil)
                }
            } else {
                completion(nil)
            }
        }.resume()
    }
```

Changed to logic to loop through the indexes and take the asset where the `name` equals `JamfCheck.dmg` and grab the download URL from that asset.

```
    func getLatestReleaseURL(completion: @escaping (String?) -> Void) {
        let apiURL = URL(string: "https://api.github.com/repos/txhaflaire/JamfCheck/releases/latest")!
        
        URLSession.shared.dataTask(with: apiURL) { data, _, error in
            if let data = data {
                do {
                    if let json = try JSONSerialization.jsonObject(with: data, options: []) as? [String: Any],
                       let assets = json["assets"] as? [[String: Any]] {
                        if let matchingAsset = assets.first(where: { asset in
                            if let name = asset["name"] as? String {
                                return name.contains("JamfCheck.dmg")
                            }
                            return false
                        }),
                        let assetURL = matchingAsset["browser_download_url"] as? String {
                            completion(assetURL)
                        } else {
                            completion(nil)
                        }
                    } else {
                        completion(nil)
                    }
                } catch {
                    completion(nil)
                }
            } else {
                completion(nil)
            }
        }.resume()
    }
    ```

**Comment by txhaflaire** [comment]
Resolved in release 2.1.0
